src/sentry/static/sentry/app/views/organizationEvents/eventsChart.jsx

```diff
@@ -34,13 +34,6 @@ class EventsLineChart extends React.Component {
       return false;
     }
 
-    if (
```

Comment:
@billyvg Was there a particular reason to cancel the render on based on the props `timeseriesData` and `releaseSeries`?

Reply:
I decided to revert and just add `isEqual(this.props.previousTimeseriesData, nextProps.previousTimeseriesData)`.